README.md

```diff
@@ -1 +1,187 @@
+- [📝 License](#license)
+
+
+
+# 📖 My Portfolio setup<a name="about-project"></a>
+
+> Created my profolio project using HTML and CSS
+
+
+
+## 🛠 Built With  <a name="built-with"></a>
+1-HTML</br>
+2-CSS</br>
+3-LINTERS</br>
+### Tech Stack <a name="tech-stack"></a>
+
+> 
+
+<details>
+  <summary>Client</summary>
+  <ul>
+    <li><a href="https://index.html/">HTML</a></li>
+  </ul>
+</details>
+
+
+
+
+
+### Key Features <a name="key-features"></a>
+
+>
+- **Added menu** [My logo and menu icon added] 
+- **Styled using flex** [This page is styled using css flex]
+
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+
+
+## 🚀 Live Demo <a name="live-demo"></a>
+
+> 
+
+- [Live Demo Link](https://google.com)
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+
+
+To get a local copy up and running, follow these steps.
+
+### Prerequisites
+
+In order to run this project you need:
+
+-Text editor</br>
+-Web browser
+
+### Setup
+
+Clone this repository to your desired folder:
+
+
+Run command: git clone git@github.com:Porkalaikarthika/demo.git
+### Install
+
+Install this project with:
+
+
+  
+  npm install
+
+
+### Usage
+
+To run the project, execute the following command:
+
+Open the index.html file
+
+
+
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+
+
+## 👥 Authors <a name="authors"></a>
+
+
+
+👤 Porkalai karthika
+
+- GitHub: [@githubporkalaikarthika](https://github.com/Porkalaikarthika)
+- Twitter: [@twitterkarthika](https://twitter.com/PorkalaiKarthi)
+- LinkedIn: [LinkedIn](linkedin.com/in/porkalai-karthika/)
+
+
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- FUTURE FEATURES -->
+
+## 🔭 Future Features <a name="future-features"></a>
+
+
+
+- Add  project links
+- Add contact forms
+- Add blogs
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+
+
+## 🤝 Contributing <a name="contributing"></a>
+
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the [issues page](../../issues/).
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- SUPPORT -->
+
+## ⭐️ Show your support <a name="support"></a>
+
+
+
+If you like this project share and comment
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- ACKNOWLEDGEMENTS -->
+
+## 🙏 Acknowledgments <a name="acknowledgements"></a>
+
+
+
+I would like to thank Microverse
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+
+## 📝 License <a name="license"></a>
+
+This project is [MIT](./LICENSE) licensed.
+
+_NOTE: we recommend using the [MIT license](https://choosealicense.com/licenses/mit/) - you can set it up quickly by [using templates available on GitHub](https://docs.github.com/en/communities/setting-up-your-project-for-healthy-contributions/adding-a-license-to-a-repository). You can also use [any other license](https://choosealicense.com/licenses/) if you wish._
+
```

Comment:
- this is only a description of how to go about customising this section, so you can remove it from your readme so it looks professional :+1: